app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -1,14 +1,285 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/userprofile_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".Fragments.Common.UserProfileFragment">
 
-    <!-- TODO: Update blank fragment layout -->
-    <TextView
+    <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:text="User Profile Fragment" />
+        android:layout_alignParentTop="true"
+        android:layout_alignParentBottom="true"
+        android:layout_marginLeft="20dp"
+        android:layout_marginRight="20dp"
+        android:scrollbars="none">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <!-- Header Layout -->
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/back"
```

Comment:
Like i mentioned earlier, for ids can we just make it slightly more descriptive. Maybe we don't need the classname in it, but something i like is to append the type of element at the end (ie backTV) or if this was a button (backBtn). 
To make it even more unique though it would be good to have something like 'userProfileBackBtn', and this prevents and chance of someone else trying to use the same ID across the app.